shell/platform/darwin/ios/BUILD.gn

```diff
@@ -69,7 +69,7 @@ source_set("flutter_framework_source") {
   cflags_objc = flutter_cflags_objc
   cflags_objcc = flutter_cflags_objcc
 
-  deps = []
+  deps = [ "//flutter/third_party/spring_animation" ]
```

Comment:
Better to move in below collection in `source_set("flutter_framework_source")` ?:

```gn
deps += [
    ":ios_gpu_configuration",
    "//flutter/common",
    "//flutter/common/graphics",
    "//flutter/flow",
    "//flutter/fml",
    "//flutter/lib/ui",
    "//flutter/runtime",
    "//flutter/runtime:libdart",
    "//flutter/shell/common",
    "//flutter/shell/platform/common:common_cpp_input",
    "//flutter/shell/platform/darwin/common",
    "//flutter/shell/platform/darwin/common:framework_shared",
    "//flutter/shell/platform/embedder:embedder_as_internal_library",
    "//flutter/shell/profiling:profiling",
    "//third_party/skia",
  ]

```